includes/WikiSEO.php

```diff
@@ -156,7 +156,9 @@ public function setMetadata( array $metadataArray, ParserOutput $out = null ): v
 	 * @param OutputPage $out
 	 */
 	public function addMetadataToPage( OutputPage $out ): void {
-		$this->modifyPageTitle( $out );
+		if ( $out->getPageTitle() === $out->getDisplayTitle() ) {
```

Comment:
```suggestion
		if ( $out->isArticle() ) {
```
We can check for `isArticle` which returns true on view action